os/include/tinyara/pm/pm.h

```diff
@@ -297,13 +297,18 @@ enum pm_state_e {
 	PM_COUNT,
 };
 
+struct timer_s {
```

Comment:
The name is too common, let me suggest `struct pm_wakeup_timer_s`

Reply:
Sure, will change accordingly~

Reply:
This change is still pending...